compiler/importer.nim

```diff
@@ -344,9 +344,11 @@ proc evalImport*(c: PContext, n: PNode): PNode =
           imp[lastPos] = x[1]
           impAs[1] = imp
           impAs[2] = x[2]
+          impAs.info = x[2].info
```

Comment:
Shouldn't this be `x[1].info`?

Reply:
Probably, but
```nim
inport unused as uns
```
Will strangely warn about `uns` is not used, not about `unused`